PageObjectPattern/src/test/java/utils/TestDataGenerator.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Random;
+
+public class TestDataGenerator {
+
+	// Method to create random email address
+	public String getRandomEmail() {
+		String staticName = "ytestautomation@mailinator.com";
+		Random rand = new Random();
+		int rand_number = rand.nextInt(1000);
+		String randomEmail = rand_number + staticName;
+		return randomEmail;
+	}
+	// Method to save Email to file
+		public  void saveEmail(String userEmail) throws IOException {
```

Comment:
Do we need to have these 2 methods public? Maybe we could have one public method that we will be using in our tests and one private method.

Reply:
done